### PR TITLE
Fix issue #24

### PR DESCRIPTION
**Pull request by hello-ashleyintech**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [x] Bug fix
- [ ] Documentation

### Summary

fixes issue https://github.com/slack-samples/deno-simple-survey/issues/24 and also updated README for extra selecting auth step to select a specific account for the workflow.

Will also make an update on the tutorial side as well.

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

**Comment by hello-ashleyintech** [comment]
TODO for @hello-ashleyintech : Once this code change is approved, before merging this, modify tutorial on api.slack.com to include the updated snippets as well as the `select-auth` step

**Comment by rampion** [comment]
Also fixes issue #8

File: functions/save_response.ts [hunk 1]
```diff
@@ -31,7 +31,6 @@ export const SaveResponseFunctionDefinition = DefineFunction({
       "reactor_access_token_id",
       "google_spreadsheet_id",
       "impression",
-      "comments",
```
Comment (zimeg) [inline functions/save_response.ts:34]: 👋 farewell, required `comments`!

File: README.md [hunk 2]
```diff
@@ -169,6 +169,20 @@ $ slack external-auth add
 Once you've successfully connected your account, you're almost ready to create
 surveys at the press of a reaction!
 
+To complete the connection process, you need to let your app know you'll be
+using the account you just authenticated with. To do so, run:
```
Comment (zimeg) [inline README.md:173]: Feel free to ignore, but I'm wondering if it'd be helpful to be more explicit here? Maybe saying this is to select the authentication to use for this specific workflow?

**Review by zimeg — APPROVED** [review]
Two great fixes here! Thanks for patching both and catching the `select-auth` step :clap: :clap:

This is already polished and works wonderfully so giving this an approval! I left a small comment to the wording of the README but this is pretty minor. It reads well as is so feel free to merge whenever!

**Comment by hello-ashleyintech** [comment]
Added in tutorial update - should reflect in the docs site in the next few hours. Merging this in now ✅

**Pull request merged**